src/sentry/api/endpoints/project_trace_item_details.py

```diff
@@ -88,10 +111,10 @@ def get(request: Request, project: Project, item_id: str) -> Response:
         resp_dict = {
             "itemId": resp["itemId"],
             "timestamp": resp["timestamp"],
-            "attributes": {
-                attr["name"]: convert_rpc_attribute_to_json(attr["value"])
-                for attr in resp["attributes"]
-            },
+            "attributes": sorted(
+                list(convert_rpc_attribute_to_json(resp["attributes"], trace_item_type)),
```

Comment:
nit: Would it be cleaner to just have `convert_rpc_attribute_to_json` just return the sorted list? (actually especially given the name too)

Reply:
yep yep